app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
Discussion was continued in https://hackclub.slack.com/archives/C3P3J5N9J/p1492133698713316 for future reference.